src/test/java/mops/businesslogic/PermissionServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package mops.businesslogic;
+
+import mops.exception.MopsException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Set;
+
+import static mops.businesslogic.PermissionServiceProdImpl.URL;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PermissionServiceTest {
```

Comment:
```suggestion
class PermissionServiceTest {
```

Reply:
why not public?

Reply:
because they don't need to be public. every test class is package-private